docs/howto/README.md

```diff
@@ -14,8 +14,6 @@ quickly accomplish common tasks.
 - [Running Krustlet on any Kubernetes cluster with
   inlets](krustlet-with-inlets.md)
 - [Running Krustlet on MicroK8s](krustlet-on-microk8s.md)
-- [Running Kubernetes on Amazon Elastic Kubernetes Service
```

Comment:
Did you mean to remove this?

Reply:
yes -- it's a dupe of the 2nd line with a different name :) 

not picking on AWS, I promise!!

Reply:
actually, just realized there's a dupe of Kind too

Reply:
Ok sweet!